client-src/default/utils/reloadApp.js

```diff
@@ -13,7 +16,12 @@ function reloadApp(
   }
   if (hot) {
     log.info('[WDS] App hot update...');
-    const hotEmitter = require('webpack/hot/emitter');
+    const hotEmitter =
+      typeof __webpack_hot_emitter__ !== 'undefined'
+        ? // eslint-disable-next-line no-undef
+          __webpack_hot_emitter__
```

Comment:
I had to change slightly how the hot emitter is imported, because it is important that the right hot emitter module is used for hot reloading to work.